src/executor/Scheduler.ts

```diff
@@ -418,6 +418,7 @@ export class Scheduler {
         this.plugins.forEach((plugin) => {
             if (
                 plugin.hasCompressCssAssets &&
+                !plugin.hasVendors &&
```

Comment:
@slaven3kopic in this method all plugins that have `import.css` will be registered for the compress css task. Additionally, if there are vendors in the plugin, the same task will be executed again (called from the vendors task). Since having vendors and `import.css` file are independent of each other, this was the simplest approach to handle it.

Reply:
@vladimir-arsov Did you test if importing the LESS/SCSS/CSS in `index.ts` still works?

Reply:
@slaven3kopic from what i tested, all the stuff work as before.